google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractResultSet.java

```diff
@@ -490,6 +501,7 @@ private static Object decodeValue(Type fieldType, com.google.protobuf.Value prot
           checkType(fieldType, proto, KindCase.BOOL_VALUE);
           return proto.getBoolValue();
         case INT64:
+        case ENUM:
```

Comment:
Please double check if ENUM should be Kindcase.STRING_VALUE, or some other value such as `NUMBER_VALUE` from https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/Value.KindCase

Reply:
Rechecked: Wire type of Enum and Int64 is same, so should be wrapped as STRING_VALUE.